sdk/batch/Azure.Compute.Batch/src/Generated/Docs/BatchClient.xml

```diff
@@ -63,7 +63,21 @@ Uri endpoint = new Uri("<endpoint>");
 TokenCredential credential = new DefaultAzureCredential();
 BatchClient client = new BatchClient(endpoint, credential);
 
-BatchPoolCreateContent pool = new BatchPoolCreateContent(null, null);
+BatchPoolCreateContent pool = new BatchPoolCreateContent("mypool01", "Standard_D1_v2")
```

Comment:
these samples were removed in a regression in TCGC's 0.50.2, now they are fixed therefore we are getting them back.